src/cli/cmds/synth.ts

```diff
@@ -14,32 +15,61 @@ class Command implements yargs.CommandModule {
   public readonly builder = (args: yargs.Argv) => args
     .option('app', { default: config.app, required: true, desc: 'Command to use in order to execute cdk8s app', alias: 'a' })
     .option('output', { default: config.output, required: false, desc: 'Output directory', alias: 'o' })
-    .option('stdout', { type: 'boolean', required: false, desc: 'Write synthesized manifests to STDOUT instead of the output directory', alias: 'p' });
+    .option('stdout', { type: 'boolean', required: false, desc: 'Write synthesized manifests to STDOUT instead of the output directory', alias: 'p' })
+    .option('plugins-dir', { default: config.pluginsDirectory, required: false, desc: 'Directory to store cdk8s plugins.' })
+    .option('validate', { type: 'boolean', default: true, required: false, desc: 'Apply validation plugins on the resulting manifests (use --no-validate to disable)' });
+  ;
 
   public async handler(argv: any) {
+
     const command = argv.app;
     const outdir = argv.output;
     const stdout = argv.stdout;
+    const validate = argv.validate;
+    const pluginsDir = argv.pluginsDir;
 
-    if (outdir !== config.output && stdout) {
+    if (outdir && outdir !== config.output && stdout) {
       throw new Error('\'--output\' and \'--stdout\' are mutually exclusive. Please only use one.');
     }
 
-    await fs.remove(outdir);
+    if (outdir) {
+      await fs.remove(outdir);
+    }
```

Comment:
FYI - you typically don't need `fs-extras` anymore with Node 14+...

Here, for example, if you want "recursive, idempotent rm", you can:

```suggestion
    await fs.rm(outdir, { force: true, recursive: true });
```

Reply:
Nice. Thanks.